bids/layout/db.py

```diff
@@ -107,7 +107,6 @@ def reset_database(self, init_args, config=None):
             layout_info = LayoutInfo(**init_args)
         self.session.add(layout_info)
         # Add config records
-        config = listify('bids' if config is None else config)
```

Comment:
If we don't want to use `['bids']` as a default, we should make `config` a required argument. Otherwise this is just going to cause the next line to fail if `reset_database()` is called without a config.